src/loop_libraries/loop_library.cpp

```diff
@@ -109,7 +109,7 @@ void Loop_library::set(int new_type)
    }
 }
 
-Loop_library::Library Loop_library::get_type()
+const Loop_library::Library Loop_library::get_type()
```

Comment:
Sorry, I my proposal was probably confusing. :)
Please make the *function* (i.e. the `this` pointer) `const`, not the return type, such that the function can be called on `const` objects etc.:
~~~
Loop_library::Library Loop_library::get_type() const
~~~

Reply:
@Expander, this function is a _static_ one. It seems, that static const functions are not allowed in C++. This is why I thougth, that you meant to make the _return_ type const.

Reply:
Ouch! Sorry for my stupidity, I missed that. :) (The `type` it returns confused me, but it is also `static`.) Please ignore my comment and revert commit a8903e3 .